### PR TITLE
weather API complete

### PR DESCRIPTION
**Pull request by MarcusSchnake**

### CODE REVIEW

File: team-git/src/apis/Weather.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            const getLat = (pos) => setLat(pos.coords.latitude)
+            const latitude = navigator.geolocation.getCurrentPosition( getLat );  
+        }  
+        if(lng === "") {
+            const getLng = (pos) => setLng(pos.coords.longitude)
+            const longitude = navigator.geolocation.getCurrentPosition( getLng );
+        }
+    }, [lat, lng])
+
+    // console.log(lat, lng)
```
Comment (LA-Miller) [inline team-git/src/apis/Weather.js]: remove this
```suggestion
```

File: team-git/src/apis/Weather.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    })
+
+    useEffect(() => {
+        if (currentWeather.length < 1 && lat && lng) {
+            weatherData();
+            console.log(weatherData);
+        }
+    }, [currentWeather, lat, lng]);
+
+    console.log(weatherDescription)
```
Comment (LA-Miller) [inline team-git/src/apis/Weather.js]: remove this
```suggestion
```

File: team-git/src/App.js [hunk 3]
```diff
@@ -1,16 +1,14 @@
 import React from 'react';
 import 'bootstrap/dist/css/bootstrap.css';
 import './App.css';
-import UseGeoLocation from './geolocation/UseGeoLocation';
```
Comment (LA-Miller) [inline team-git/src/App.js:4]: Nice job mate

**Review by LA-Miller — CHANGES_REQUESTED** [review]

File: team-git/src/App.js (hunk 3 above)
Comment (MarcusSchnake) [reply team-git/src/App.js:4]: Thank you!!!

**Review by LA-Miller — APPROVED** [review]
⭐

**Pull request merged**